### PR TITLE
Update install-spring-cloud-gateway.hbs.md

### PR DESCRIPTION
**Pull request by Samze**
Fix installation namespace

# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

TAP 1.5 and beyond

### CODE REVIEW

File: spring-cloud-gateway/install-spring-cloud-gateway.hbs.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Default values
     $ tanzu package install spring-cloud-gateway \
         --package-name spring-cloud-gateway.tanzu.vmware.com \
         --version 2.0.0 \
-        --namespace spring-cloud-gateway
+        --namespace tap-install
```
Comment (csterwa) [inline spring-cloud-gateway/install-spring-cloud-gateway.hbs.md:74]: Please review SCG for K8s guide on installing on TAP for information on how to add values file to ensure that operator is not installed into `tap-install` namespace.

https://docs.vmware.com/en/VMware-Spring-Cloud-Gateway-for-Kubernetes/2.0/scg-k8s/GUID-operator-installation-tanzu-cli.html#install-the-spring-cloud-gateway-for-kubernetes-package

Comment (Samze) [reply spring-cloud-gateway/install-spring-cloud-gateway.hbs.md:74]: Probably makes sense for someone who owns Spring Docs on TAP to make this more intrusive docs change. cc @RichardJJG do you know who that may be?

Comment (csterwa) [reply spring-cloud-gateway/install-spring-cloud-gateway.hbs.md:74]: I can make the changes but my preference is to link out to the SCG for K8s docs so we don't have duplication.

Comment (RichardJJG) [reply spring-cloud-gateway/install-spring-cloud-gateway.hbs.md:74]: @fifthposition is the writer assigned to the SCG for K8s docs.

Comment (RichardJJG) [reply spring-cloud-gateway/install-spring-cloud-gateway.hbs.md:74]: @fifthposition, are you okay to make a story for the changes described above in the  SCG for K8s docs? I'll handle the one code change that Chris wants here in the TAP docs.

**Pull request merged**